src/main/java/com/github/kongchen/swagger/docgen/mavenplugin/SecurityDefinition.java

```diff
@@ -114,4 +145,12 @@ public String getIn() {
     public void setIn(String in) {
         this.in = in;
     }
+
+    public String getJson() {
```

Comment:
I need to remove these get / set methods I added. They may cause trouble somewhere else in smp.

Reply:
They are needed for the unit tests, so I will leave them here for now..